src/sbt-test/sbt-1.5/scala-3/test

```diff
@@ -0,0 +1,18 @@
+# check that we can run a syntactic rule against a Scala 3 dialect source file
+-> nocurlybraces/scalafix --check SyntacticRule
+> nocurlybraces/scalafix SyntacticRule
+> nocurlybraces/scalafix --check SyntacticRule
+
+# enable semanticdb output for Scala 3 compiler
+> scalafixEnable
+
+# check that projects supported by semanticdb-scalac can still compile
+> scala211/compile
+
+# check that projects not supported by semanticdb-scalac can still compile
+> scala210/compile
```

Comment:
this is somewhat unrelated to the scala3 scripted, but as mentioned in another comment, this is a safeguard until I restructure how scripted are ran against different sbt versions to better ensure that `scalafixEnable` works pre/post 1.3 (since the PR introduces a different behavior when `SemanticdbPlugin` is available).